### ISSUE TITLE
cold start

### ISSUE THREAD

**Issue by heiyl** [getsentry/sentry-java#2128]
### Integration

sentry-android

### Build System

Gradle

### AGP Version

6.8

### Proguard

Enabled

### Version

6.0

### Steps to Reproduce

Android正常接入sentry

### Expected Result

上报到performance中的cold start 冷启动时间只有App的第一个启动页面能统计到

### Actual Result

上报到performance中的cold start 冷启动时间目前好多Activity都能统计到，这个不符合统计文档中冷启动的定义。
![image](https://user-images.githubusercontent.com/5343411/175252202-409f2f6c-ebd0-4c1f-b4c8-d4bb65868d2d.png)

**Comment by romtsn** [comment]
hey, sorry you're having troubles. Are you using sentry self-hosted? We recently removed this feature in the latest version to avoid the confusion, could you update it?

**Comment by heiyl** [comment]
你们是在6.1.4版本关闭了上报冷启动统计的功能吗？因为这个针对启动性能的监听对我们是有用的，这个不能开启修复吗？

**Comment by romtsn** [comment]
It has nothing to do with the Android SDK version, but rather with your Sentry self-hosted version. We still report the cold/warm startup time for the first screen in your app, you can see it in multiple places:

1. On the Performance overview page, there are multiple widgets for this:
<img width="1565" alt="image" src="https://user-images.githubusercontent.com/4999776/176861779-b3cc5d21-1533-45bb-9a41-e763ba296c8e.png">

2. On the individual transaction page, there's a span and a little element on the side saying "Mobile Vitals":
<img width="1574" alt="image" src="https://user-images.githubusercontent.com/4999776/176862321-099b1dee-41d5-4d05-9d46-0299cf6b4558.png">

3. You can also select spans similat to the cold start span to see the history of app launches:
<img width="1568" alt="image" src="https://user-images.githubusercontent.com/4999776/176862577-ab94d327-cfaa-4ea8-b27c-b297e5795d2b.png">

**Comment by heiyl** [comment]
感谢您的回答，我提的这个问题是由于sentry托管的版本太低的原因吗？
因为我初始化SDK时是在Application的onCreate()中初始化的，我排查了代码，由于Sentry的初始化参数我这里是通过接口请求下发的，下发后我又进行了初始化，我发现多次初始化，导致了冷启动的标识不准确了，这里我进行了优化，只在Application的onCreate()中初始化，以避免冷启动统计有问题。

**Comment by romtsn** [comment]
Yes, you should update the self-hosted version to the latest to get the aforementioned changes

**Comment by heiyl** [comment]
好的，谢谢！
